pkg/api/helpers.go

```diff
@@ -53,7 +53,7 @@ func (c *ConversionError) Error() string {
 var Semantic = conversion.EqualitiesOrDie(
 	func(a, b resource.Quantity) bool {
 		// Ignore formatting, only care that numeric value stayed the same.
-		// TODO: if we decide it's important, after we drop v1beta1/2, we
+		// TODO: if we decide it's important, after we drop v1beta1/2(deleted on June 1, 2015), we
```

Comment:
TODO: if we decide it's important, it should be safe to start comparing the format.